server/services/caldav/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "gladys-caldav",
+  "main": "index.js",
+  "os": [
+    "darwin",
+    "linux",
+    "win32"
+  ],
+  "cpu": [
+    "x64",
+    "arm",
+    "arm64"
+  ],
+  "scripts": {},
+  "dependencies": {
+    "bluebird": "^3.7.0",
+    "dav": "git+https://github.com/bertrandda/dav.git#gladys-custom",
+    "ical": "^0.6.0",
+    "moment": "^2.24.0",
+    "request": "^2.88.0",
+    "request-promise-native": "^1.0.7",
```

Comment:
This package is deprecated, see

https://github.com/request/request-promise-native

Reply:
Also a waste from the old way.